src/main/kotlin/no/nav/medlemskap/regler/common/Funksjoner.kt

```diff
@@ -16,6 +16,8 @@ object Funksjoner {
 
     infix fun List<String>.inneholder(string: String) = this.contains(string)
 
+    infix fun List<String>.alleEr(string: String) = this.distinct().size > 1
```

Comment:
Denne gjør ingen sjekk på string/parameter som sendes inn. Den sjekker kun om listen har mer enn 1 unik verdi. Hvis formålet er å sjekke at listen ikke har andre verdier enn det som er sendt inn blir det noe i retning: this.stream().allMatch { p -> p.equals(string)}

Reply:
Takk! Lette etter en slik metode så ble kanskje en litt forenklet løsning.

Reply:
endrer det